Source/WebCore/platform/text/PlatformLocale.h

```diff
@@ -65,7 +65,7 @@ class Locale {
         RightToLeft
     };
 
-#if PLATFORM(COCOA)
+#if PLATFORM(GTK) || PLATFORM(WPE) || PLATFORM(COCOA)
```

Comment:
This breaks PlayStation, as another consumer of LocaleICU. Seems like this should just be `!PLATFORM(WIN)`.

Reply:
yeah, `!PLATFORM(WIN)` should do it. I can open a fix soon

Reply:
No worries, I've opened https://github.com/WebKit/WebKit/pull/37612.